config/application.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require_relative 'boot'
+require_relative '../lib/scholarsphere/redis_config'
```

Comment:
@whereismyjetpack this is just `require 'scholarpshere/redis_config'`

Reply:
I think i'm missing something as part of initialization, i tried that and got ```LoadError: cannot load such file -- scholarsphere/redis_config```